train.py

```diff
@@ -47,6 +47,9 @@
                   'Whether to use Bigtable as input.  '
                   '(Only supported with --use_tpu, currently.)')
 
+flags.DEFINE_bool('freeze', False,
```

Comment:
this is funny; should we plumb this into the rl_loop/ usage?

Reply:
Probably.

Assuming we pay a similar ~30 second TF init cost on TPU to what I see on my 8 GPU GCE VM, then freezing in the same binary as training would save ~8 hours in a 1000 model run.